cpp/tests/column/column_test.cu

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/column/column_view.hpp>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
```

Comment:
Can tests be made .cpp files now that we have `device_buffer` which is uninitialized?

Reply:
Done. Though occasionally I will have tests that use `thrust` calls directly in the tests file, and will need to be `*.cu` files.